src/app.js

```diff
@@ -24,6 +24,7 @@ app.use(express.static('public'))
 app.use(favicon(__dirname + '/public/favicon.ico'));
 
 // routes
-app.use('/api/v1', mainRouter, userRouter, classesRouter);
+app.use('/api/v1', mainRouter, userRouter , classesRouter);
```

Comment:
do we need to use here `userRouter , classesRouter` because they use `/api/v1/users` path on the line 28

Reply:
changed, and added letter validation, so User should enter only letters for last name and first name, and also if adultName is required, I added validation to check if it contains first AND last name